### PR TITLE
embiggen:0.0.1

### PR DESCRIPTION
**Pull request by quantum9Innovation**
I am submitting
- [x] a new package
- [ ] an update for a package

**Description**: This package implements LaTeX primitives for manual sizing of delimeters. Typst has automatic sizing of most delimeters using `lr`. This can be further customized with the `scale` parameter, but LaTeX users transitioning to Typst are probably most familiar with the `big`, `Big`, `bigg`, and `Bigg` commands for manually sizing delimeters. The embiggen package implements these functions in Typst by supplying pre-computed scale factors to the `lr` function, essentially adding aliases for common scaling factors applied to `lr`. There are also functions that make the output of `lr` smaller (`small`, `Small`, `smalll`, and `Smalll`). This package was inspired by https://github.com/typst/typst/issues/4048. 

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

**Comment by Enter-tainer** [comment]
sounds like https://typst.app/universe/package/delimitizer

File: packages/preview/embiggen/0.0.1/demo.typ [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#import "src/lib.typ": *
```
Comment (elegaanz) [inline packages/preview/embiggen/0.0.1/demo.typ:1]: This import could use the package import to make it easier to start from this example.

```suggestion
#import "@preview/embiggen:0.0.1": *
```

**Review by elegaanz — COMMENTED** [review]
Looks good! I just made a suggestion.

**Comment by quantum9Innovation** [comment]
> sounds like [typst.app/universe/package/delimitizer](https://typst.app/universe/package/delimitizer) but this package has extra small support

That and the syntax is different, since `big`, `Big`, etc. act as drop-in replacements for `lr`. Also, these commands scale the output of `lr` instead of imposing a fixed size like they do in LaTeX.

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**